tasks/main.yml

```diff
@@ -39,8 +39,13 @@
 - include: configure-Debian.yml
   when: ansible_os_family == 'Debian'
 
-- name: Ensure Apache is started and enabled on boot.
+- name: Ensure Apache is enabled on boot.
   service:
     name: "{{ apache_daemon }}"
-    state: started
```

Comment:
Using the variable as defined per comment above, this line can remain as `state: {{ apache_state }}`.